docs/Observability/2. prometheus.md

```diff
@@ -0,0 +1,20 @@
+# Prometheus method
+## Introduction
+Prometheus is an open-source monitoring and alerting toolkit designed for reliability and scalability. It collects metrics from configured targets at specified intervals, stores them in a time-series database, and provides powerful querying capabilities. With its flexible architecture, Prometheus is particularly suited for dynamic environments, making it a popular choice for cloud-native applications and microservices. Its intuitive visualization tools help teams gain deep insights into system performance, ensuring timely detection of issues.
```

Comment:
link to prometheus and prometheus operator

Reply:
Done